modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -507,7 +507,13 @@
          "PROCESSED": "Finished",
          "RECORDING_FAILURE": "Recording failure",
          "PROCESSING_FAILURE": "Processing failure",
-         "PROCESSING_CANCELED": "Processing canceled"
+         "PROCESSING_CANCELED": "Processing canceled",
+         "RETRACTING": "Retracting",
+         "DUPLICATING": "Duplicating",
+         "PUBLISHING": "Publishing",
+         "IMPORTING": "Importing",
+         "REPUBLISHING": "Republishing",
+         "UPDATING_PREVIEWS": "Updating previews"
```

Comment:
As @krinnewitz has not yet answered: From the point of view of the Admin UI it does not really matter whether the string to be translated is an actual event status or displayable event status. It's always correct to just use those translation keys (for translating both the "actual" event statuses and "displayable" event statuses).

Why would you want too separate them? Seems to make thing more complicated...?

Reply:
It might not matter for the Admin UI, but there is a difference between those two kinds, and when defining state mappings for workflows, it might be nice to be able to differentiate between the general statuses and the specific ones. I don't think this makes it more complicated, on the contrary I think it makes things clearer.